### PR TITLE
Bump observability-bundle

### PR DESCRIPTION
**Pull request by QuentinBisson**
### Checklist
- [ ] Roadmap issue created
- [ ] Release uses latest stable Flatcar
- [ ] Release uses latest Kubernetes patch version

### CODE REVIEW

File: aws/requests.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ releases:
 - name: ">= 20.0.0"
   requests:
   - name: observability-bundle
-    version: ">= 1.1.0"
+    version: ">= 1.2.2"
```
Comment (hervenicol) [inline aws/requests.yaml:5]: New release out! :grin: 
```suggestion
    version: ">= 1.2.3"
```

**Comment by QuentinBisson** [comment]
@giantswarm/team-phoenix can you check this please ?

**Review by hervenicol — APPROVED** [review]
LGTM, but shouldn't it be pushed to other providers as well?

**Comment by QuentinBisson** [comment]
Nope, we only have aws now in vintage

**Pull request merged**